segmentstore/server/src/test/java/io/pravega/segmentstore/server/containers/StreamSegmentContainerTests.java

```diff
@@ -2158,7 +2158,7 @@ private void mergeTransactions(HashMap<String, ArrayList<String>> transactionsBy
         Futures.allOf(mergeFutures).get(TIMEOUT.toMillis(), TimeUnit.MILLISECONDS);
     }
 
-    private RefCountByteArraySegment getAppendData(String segmentName, int appendId) {
+    public static RefCountByteArraySegment getAppendData(String segmentName, int appendId) {
```

Comment:
Why do you need this? I don't see where you use this or why you need a `RefCountByteArraySegment`. That class is meant to track append leaks, which you do not need to do.

All you need to do in your tests is generate a basic `ByteArraySegment`, so you don't need these changes here.

Reply:
Fixed.